Dashboard/src/components/Infrastructure.js

```diff
@@ -14,10 +14,10 @@ class Infrastructure extends Container {
     let servers, workstations, dns;
 
     if (this.state.data) {
-      const upSort = (a, b) => a.Status === 'up';
-      servers = this.state.data.Servers.sort(upSort);
-      workstations = this.state.data.Workstations.sort(upSort);
-      dns = this.state.data.DNS;
+      const upSort = (a, b) => a.status === 'up';
+      servers = this.state.data.servers.sort(upSort);
+      workstations = this.state.data.workstations.sort(upSort);
+      dns = this.state.data.dns;
```

Comment:
Sort is an in-place algorithm, you probably need to use spread operators here first.
see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort

Reply:
See above note about #65.